financial-connections/src/main/java/com/stripe/android/financialconnections/repository/FinancialConnectionsManifestRepository.kt

```diff
@@ -596,6 +598,19 @@ private class FinancialConnectionsManifestRepositoryImpl(
         )
     }
 
+    private fun updateIDConsentContentPane(
+        source: String,
+        pane: IDConsentContentPane?
+    ) {
+        logger.debug("SYNC_CACHE: updating local sync object from $source with ID consent content pane")
+        val cachedResponse = cachedSynchronizeSessionResponse
+        cachedSynchronizeSessionResponse = cachedResponse?.copy(
+            text = cachedResponse.text?.copy(
+                idConsentContentPane = pane,
+            ),
+        )
+    }
```

Comment:
`/institution_selected` returns a partial `text` object, so we need to keep the original fields in it.